pkg/restapi/vc/operation/operations.go

```diff
@@ -243,14 +243,12 @@ func (o *Operation) storeVCHandler(rw http.ResponseWriter, req *http.Request) {
 	doc.Content["message"] = data
 	doc.ID = data.Credential.ID
 
-	locationOfDocument, err := o.client.CreateDocument(data.Profile, &doc)
+	_, err = o.client.CreateDocument(data.Profile, &doc)
```

Comment:
I changed this because I don't think the caller should or wants to know about the EDV location - that's a detail internal to vc-service.